examples/cloudflare-durable-object-websocket/alchemy.run.ts

```diff
@@ -34,11 +34,11 @@ export const client = await Vite("client", {
 
 console.log("Client:", client.url);
 
-await app.finalize();
-
 if (process.env.ALCHEMY_E2E) {
   const { test } = await import("./test/e2e.ts");
   await test({
     url: server.url,
   });
 }
+
+await app.finalize();
```

Comment:
Calling `app.finalize()` first meant the E2E test wouldn't actually run because the process would exit on finalize